Glossary.adoc

```diff
@@ -1,5 +1,17 @@
 // Here goes all the term definitions the user might need to reference while reading the manual.
-// We need to figure out a way to sort the items here alphabetically
-// https://github.com/zynaddsubfx/user-manual/issues/6
 
-NOTE: Here by glossary!
\ No newline at end of file
+// Filter, sync, distortion, fade, pan
+// LFO, FM, AM, Bandwith, Additive/Subtractive synthesis, ADSR, Envelope and Harmonic
+
+// Ideally they're more naturally introduced in the text
+
+[glossary]
+Key:: A possible input to Zyn-Fusion, with a specific notation (eg. A4, G#2, etc). Practically, it's the same thing as a note.
```

Comment:
I'm not proposing a change, but for elsewhere in the docs be aware that this is an overloaded term and may be used to talk about a whole scale in the microtonal sections of the app.